xc/device.ml

```diff
@@ -1845,6 +1849,14 @@ module Backend = struct
       val qemu_media_change : xs:Xenstore.Xs.xsh -> device -> string -> string -> unit
     end
 
+    (** Vcpu functions that use the dispatcher to choose between different profile backends *)
+    module Vcpu: sig
+      val add : xs:Xenstore.Xs.xsh -> devid:int -> int -> bool -> unit
+      val set : xs:Xenstore.Xs.xsh -> devid:int -> int -> bool -> unit
+      val del : xs:Xenstore.Xs.xsh -> devid:int -> int -> unit
+      val status : xs:Xenstore.Xs.xsh -> devid:int -> int -> bool
```

Comment:
Is `status` the best name? How about one of 

* `enabled`, `is_enabled`
* `plugged`, `is_plugged`

Reply:
'status' was part of the original interface in device.ml/mli and used from xenops_server_xen.ml:1170, so I kept it as it was:

  `  xc/xenops_server_xen.ml:1170:  do if Device.Vcpu.status ~xs ~dm:(dm_of ~vm) ~devid:i domid then n := i`

do you think we should change this name to improve is readability? If so, I would suggest the name `is_online`, since that's what is being tested in device.ml:801.

Reply:
I'd like to hear @robhoes's opinion. I think the name could be improved but also don't want to introduce confusing inconsistencies.

Reply:
"Status" is a little vague indeed. But the most important thing to me is to be consistent. So if we use "status" somewhere else in xenopsd for the same thing, then we should either change that to a better name as well, or stick with "status".

Reply:
I looked around the xenopsd files, and I couldn't spot any pattern: each vbd, vif, usp, pci etc device seems to have its own plugged, active, enabled, online etc way of stating that the device is available and can be used. For vcpus, online seems to be the jargon. If no one has any strong opinions about it, I would just leave this name as it currently is, so as to minimise changes.